clientlibs/java/src/main/java/org/upgradeplatform/utils/Utils.java

```diff
@@ -20,16 +19,16 @@ public class Utils
 	public static final String INVALID_FEATURE_FLAG_KEY= "Feature flag with given key not found";
 	public static final String INVALID_METRIC_META_DATA= "Invalid metadata type provided. It should be of MetricMetaData enum type";
 
-
-	public static final String INITIALIZE_USER= "api/v5/init";
-	public static final String SET_GROUP_MEMBERSHIP= "api/v5/groupmembership";
-	public static final String SET_WORKING_GROUP= "api/v5/workinggroup";
-	public static final String GET_ALL_EXPERIMENTS= "api/v5/assign";
-	public static final String MARK_EXPERIMENT_POINT= "api/v5/mark";
-	public static final String GET_ALL_FEATURE_FLAGS= "api/v5/featureflag";
-	public static final String LOG_EVENT= "api/v5/log";
-	public static final String SET_ALT_USER_IDS= "api/v5/useraliases";
-	public static final String ADD_MATRIC= "api/v5/metric";
+	private static final String apiVersion = "api/v5";
```

Comment:
as discussed, will need to be `v6` once the v6 client controller pr is merged in.

Reply:
changed to v6. also removed a redundant request object